projects/extension/src/background/index.ts

```diff
@@ -35,7 +35,7 @@ const init = async () => {
       const rpcCallback = (rpc: string) => {
         console.warn(`Got RPC from ${key} dummy chain: ${rpc}`);
       };
-      await manager.addChain(key, value, rpcCallback)
+      await manager.addChain(0, key, value, rpcCallback)
```

Comment:
Note that this could directly be done in the constructor of the `ConnectionManager` or in `initSmoldot`. This would avoid passing this `0` that makes no sense.
The three relay chains also don't need to be put in `#networks`.

Reply:
IMO the three relay chains should be in the networks - cause the Options page of the extension is "reading" from `#networks`. When no app is on and user goes to that page - he needs to see the "default" chains that exist in smoldot.
I have removed the 0 and default adding it inside `addChain` when `tabId` is not provided.